.github/workflows/push-docker-image.yml

```
@@ -43,7 +43,7 @@ jobs:
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
           ECR_REPOSITORY: ${{ matrix.cxName }}
-          IMAGE_TAG: ${{ github.sha }}
+          IMAGE_TAG: v${{ github.sha }}
```

no need to do this as it's a docker image version, not the version of the server I think!
```suggestion
          IMAGE_TAG: ${{ github.sha }}
```